async-voter.js

```diff
@@ -3,156 +3,168 @@ const request = require('request')
 const clientId = process.env.CLIENT_ID
 const clientSecret = process.env.CLIENT_SECRET
 
-module.exports = (app) => {
+module.exports = (app, repository) => {
 
-    let repository = new Map()
-
-    app.get('/', (req, res) => {
+  app.get('/', (req, res) => {
         res.render('index', { client_id: clientId })
-    })
-
-    app.get('/oauth', (req, res) => {
-        if (!req.query.code) {
-            res.status(500)
-            res.send({"Error": "Looks like we're not getting code."})
+  })
+
+  app.get('/oauth', (req, res) => {
+    if (!req.query.code) {
+      res.status(500)
+      res.send({'Error': "Looks like we're not getting code."})
```

Comment:
@Yorkshireman  Good catch! Would it be possible to open a new issue about improving the user experience during the app installation?